Hex2Ascii: Missing deps

Notepadqq: `1.4.8` (QtWebKit)
Node: `0.12.18`
Qt: `5.11`

```
module.js:338
    throw err;
          ^
Error: Cannot find module 'deasync'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at MessageInterpreter.invokeApi (/home/scx/.var/app/com.notepadqq.Notepadqq/config/Notepadqq/extensions/com.lintwurm.hex2ascii/node_modules/notepadqq-api/lib/message_interpreter.js:25:13)
    at Notepadqq.Stub._methodMissing (/home/scx/.var/app/com.notepadqq.Notepadqq/config/Notepadqq/extensions/com.lintwurm.hex2ascii/node_modules/notepadqq-api/lib/stubs.js:25:45)
    at <error: illegal access>
    at NotepadqqApi.onWindowInitialization (/home/scx/.var/app/com.notepadqq.Notepadqq/config/Notepadqq/extensions/com.lintwurm.hex2ascii/node_modules/notepadqq-api/lib/notepadqq_api.js:69:41)
    at /home/scx/.var/app/com.notepadqq.Notepadqq/config/Notepadqq/extensions/com.lintwurm.hex2ascii/main.js:21:6
    at /home/scx/.var/app/com.notepadqq.Notepadqq/config/Notepadqq/extensions/com.lintwurm.hex2ascii/node_modules/notepadqq-api/lib/notepadqq_api.js:28:51
```

Temporary patch:
```
diff --git a/hex-ascii/package.json b/hex-ascii/package.json
index 31e62b2..e1c5550 100644
--- a/hex-ascii/package.json
+++ b/hex-ascii/package.json
@@ -2,7 +2,10 @@
   "name": "Hex2Ascii",
   "version": "0.0.1",
   "dependencies": {
-    "notepadqq-api": "^0.9.5"
+    "notepadqq-api": "^0.9.5",
+    "bindings": "~1.2.1",
+    "deasync": "^0.1.0",
+    "nan": "^2.0.7"
   },
   "private": true
 }
```

PR is on the way.

--- a/hex-ascii/package.json
+++ b/hex-ascii/package.json
@@ -2,7 +2,10 @@
   "name": "Hex2Ascii",
   "version": "0.0.1",
   "dependencies": {
-    "notepadqq-api": "^0.9.5"
+    "notepadqq-api": "^0.9.5",
+    "bindings": "~1.2.1",
+    "deasync": "^0.1.0",
+    "nan": "^2.0.7"
   },
   "private": true
 }

